docs/sphinx/using/examples/multi_gpu_workflows.rst

```diff
@@ -0,0 +1,269 @@
+Multi-GPU Workflows
+===================
+
+There are many backends available with CUDA-Q which enable seamless
+switching between GPUs, QPUs and CPUs and also allow for workflows
+involving multiple architectures working in tandem.
+
+.. code:: python
+
+    import cudaq
+    
+    targets = cudaq.get_targets()
+    
+    # for target in targets:
```

Comment:
It will be nice to actually enable this code and show the printed targets in the output.